platforms/android/tangram/build.gradle

```diff
@@ -1,6 +1,31 @@
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
 
+ext {
+  bintrayRepo = 'maven'
+  bintrayName = 'tangram'
+
+  publishedGroupId = 'com.mapzen.tangram'
+  libraryName = 'tangram-es'
+  artifact = 'tangram-es'
+
+  libraryDescription = '2D and 3D map renderer using OpenGL ES'
+
+  siteUrl = 'https://github.com/tangrams/tangram-es'
+  gitUrl = 'https://github.com/tangrams/tangram-es.git'
+
+  libraryVersion = '0.9.1'
+
+  developerId = 'tallytalwar  '
```

Comment:
jcenter gradle plugin requires developerID information. I have to put my details right now. But maybe we should look at creating tangram-es's team developer id. Letting it like this for the time being though.